lib/wallet/src/Cardano/Wallet/Read/Primitive/Tx/Mary.hs

```diff
@@ -98,11 +98,11 @@ fromMaryTx tx =
     , anyEraCerts certs
     , assetsToMint
     , assetsToBurn
-    , Just (fromLedgerTxValidity ttl)
+    , Just $ afterShelleyValidityInterval val
     )
   where
     SL.Tx bod wits mad = tx
-    MA.TxBody ins outs certs wdrls fee ttl _upd _adh mint = bod
+    MA.TxBody ins outs certs wdrls fee val _upd _adh mint = bod
```

Comment:
As the word "validity" is overloaded a lot, I think that `ttl` is a better name than `val`, even though it is slightly wrong. (The validity here is about a time interval in which the transaction is considered applicable to the ledger state.)

Just a nitpick though, happy to merge without this change.